ebs/ebs.go

```diff
@@ -303,6 +303,12 @@ func (d *Driver) CreateVolume(req Request) error {
 			log.Debugf("Failed to update tags for volume %v, but continue", volumeID)
 		}
 		volume.PreExist = true
+
+		// detach from all instances
+		for _, attachment := range ebsVolume.Attachments {
```

Comment:
This cannot be blindingly done. If the volume is being used in other VM, it will corrupt the data. It's better to think this through and have some kind of safeguard around.